ansible/roles/jenkins-master/tasks/main.yaml

```diff
@@ -73,21 +74,24 @@
 - name: Create Jenkins admin user directory
   file:
     path="{{jenkins_home_dir}}/users/{{jenkins_admin_user}}"
-    owner=jenkins group=jenkins mode=0755 state=directory recurse=yes
+    owner={{user_name}} group={{user_name}} mode=0755 state=directory recurse=yes
   tags:
     - setup
 
 - name: Create Jenkins admin user
   template:
     src=admin_config.xml.j2 dest="{{jenkins_home_dir}}/users/{{jenkins_admin_user}}/config.xml"
     force=no
+    owner={{user_name}} group={{user_name}}
   register: jenkins_admin_config
   tags:
     - setup
 
 - name: Create Jenkins config
   copy: src=config.xml dest="{{jenkins_home_dir}}/config.xml"
   register: jenkins_config
+  owner: "{{user_name}}" 
+  group: "{{user_name}}"
```

Comment:
Are we sure this is working?  I've got the impression this owner and group are misplaced.